### PR TITLE
CCCP-3385: Add new hpc.5 flavors.

### PR DESCRIPTION
**Pull request by miloudbagaa**

### CODE REVIEW

File: docs/cloud/pouta/vm-flavors-and-billing.md [hunk 1]
```diff
@@ -205,6 +209,16 @@ flavors have faster CPUs and no overcommitment of CPU cores.
...
+**hpc.5.\*:**
+
+-   No redundant power
+-      CPU:   AMD EPYC 7702 64-Core Processor,
+    ***hyper-threading***
+-   Network: Redundant 25 Gb/s
+-   Flavor disk: Stored in the central storage
+-   Single-node or disk failures may cause downtime, but instances
+    are recoverable.
+
```
Comment (Aegil) [inline docs/cloud/pouta/vm-flavors-and-billing.md:221]: These are Raid1 backed. Shouldnt cause downtime in this case.

Also these are without hyperthreading.

File: docs/cloud/pouta/vm-flavors-and-billing.md [hunk 2]
```diff
@@ -205,6 +209,15 @@ flavors have faster CPUs and no overcommitment of CPU cores.
 
 **cPouta HPC flavor characteristics:**
 
+**hpc.5.\*:**
+
+-   No redundant power
```
Comment (junousi) [inline docs/cloud/pouta/vm-flavors-and-billing.md:214]: There is redundant power. Please update

- this text statement
- the circle icons in the table (I think "p0" needs to change to "p100" or something like that)

**Review by junousi — CHANGES_REQUESTED** [review]

File: docs/cloud/pouta/vm-flavors-and-billing.md (hunk 2 above)
Comment (miloudbagaa) [reply docs/cloud/pouta/vm-flavors-and-billing.md]: This issues have been fixed by a new commit.

**Review by junousi — APPROVED** [review]

**Comment by junousi** [comment]
LGTM

File: docs/cloud/pouta/vm-flavors-and-billing.md [hunk 3]
```diff
@@ -205,6 +209,15 @@ flavors have faster CPUs and no overcommitment of CPU cores.
 
 **cPouta HPC flavor characteristics:**
 
+**hpc.5.\*:**
+
+-   Redundant power
+-      CPU:   AMD EPYC 7702 64-Core Processor,
```
Comment (junousi) [inline docs/cloud/pouta/vm-flavors-and-billing.md:215]: Ah, missed this one. This is getting rendered strangely in github, could you fix this and remove the tailing "," as well. Can also fix the indent for hpc.4 at the same time.

Comment (miloudbagaa) [reply docs/cloud/pouta/vm-flavors-and-billing.md]: The issue is fixed

**Pull request merged**